onnxruntime/core/providers/cpu/tensor/slice.cc

```diff
@@ -291,7 +291,11 @@ Status SliceImpl(OpKernelContext* ctx,
 
   SliceIterator<T> input_iterator(input_tensor, starts, output_dims, steps);
   while (output != output_end)
-    *output++ = *input_iterator++;
+  {
```

Comment:
main change

Reply:
clang should format the opening bracket on the same line?

---
In reply to: [274644523](https://github.com/Microsoft/onnxruntime/pull/820#discussion_r274644523) [](ancestors = 274644523)